.eslintrc.yaml

```diff
@@ -52,6 +52,7 @@ rules:
   no-path-concat: off
   no-proto: error
   no-redeclare: error
+  no-restricted-globals: [ error, context ]
```

Comment:
Not sure I follow, @spalger. Are you thinking of restricting other globals, or of writing this rule differently?